cloudferrylib/os/image/glance_image.py

```diff
@@ -255,16 +255,16 @@ def get_resp(data):
 
     def get_ref_image(self, image_id):
         try:
-            # ssl.ZeroReturnError happens because a size of an image is zero
+            # Skip any errors of SSL
```

Comment:
Please change comment to something meaningful, currently it simply repeats what's already written in the code